get-started/part2.md

```diff
@@ -8,76 +8,89 @@ description: Learn how to create a Docker image by writing a Dockerfile, and use
-3. Test, share and deploy your complete containerized application. 
+2. Assemble your containers and supporting infrastructure into a complete application.
+3. Test, share, and deploy your complete containerized application.
 
 In this stage of the tutorial, let's focus on step 1 of this workflow: creating the images that our containers will be based on. Remember, a Docker image captures the private filesystem that our containerized processes will run in; we need to create an image that contains just what our application needs to run.
 
 > **Containerized development environments** are easier to set up than traditional development environments, once you learn how to build images as we'll discuss below. This is because a containerized development environment will isolate all the dependencies your app needs inside your Docker image; there's no need to install anything other than Docker on your development machine. In this way, you can easily develop applications for different stacks without changing anything on your development machine.
 
-## Setting Up
+## Set up
 
-1.  Clone an example project from GitHub (if you don't have git installed, see the [install instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) first):
+Clone an example project from GitHub (if you don't have git installed, see the [install instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) first):
 
-    ```shell
-    git clone -b v1 https://github.com/docker-training/node-bulletin-board
-    cd node-bulletin-board/bulletin-board-app
-    ```
+```shell
+git clone -b v1 https://github.com/docker-training/node-bulletin-board
+cd node-bulletin-board/bulletin-board-app
+```
 
-    This is a simple bulletin board application, written in node.js. In this example, let's imagine you wrote this app, and are now trying to containerize it.
+This is a simple bulletin board application, written in Node.js. In this example, let's imagine you wrote this app, and are now trying to containerize it.
 
-2.  Have a look at the file called `Dockerfile`. Dockerfiles describe how to assemble a private filesystem for a container, and can also contain some metadata describing how to run a container based on this image. The bulletin board app Dockerfile looks like this:
+## Define a container with Dockerfile
 
-    ```dockerfile
-    FROM node:6.11.5    
+Take a look at the file called `Dockerfile` in the bulletin board application. Dockerfiles describe how to assemble a private filesystem for a container, and can also contain some metadata describing how to run a container based on this image. The bulletin board app Dockerfile looks like this:
 
-    WORKDIR /usr/src/app
-    COPY package.json .
-    RUN npm install    
-    COPY . .
+```dockerfile
+# Use the official image as a parent image
+FROM node:6.11.5
```

Comment:
we should update the `6.11.5` version everywhere.

The dockersamples PR's now use `current-slim` as tag which is version `12.14.1` at the time of writing, the current LTS version for Node.js.

Reply:
Fixed.